src/content/docs/en/installation/getting.mdx

```diff
@@ -7,7 +7,7 @@ import Alert from "../../../../components/Docs/Alert.astro";
 
 ## System Requirements
 
-If your computer could run Windows 7 well, chances are it'll run Ultramarine Linux. You should check the [System Requirements](/en/hardware/requirements) to make sure
+If your computer could run Windows 8.1, chances are it'll run Ultramarine Linux. You should check the [System Requirements](/en/hardware/requirements) to make sure

```

Comment:
So before 2024, computers that support Windows 7 can run UM, and after 2024 they magically don't? That doesn't make sense.

Reply:
i'm rethinking that for editions other than flagship and xfce, ideally we'd want to push people to newer devices if we plan to cut bios support, we can go back to windows 7 but things are only gonna get worse for those systems in newer GNOME and (to a lesser extent) Plasma versions

Reply:
you are absolutely right… but no distros currently are cutting support for BIOS, and I can't be dared to be the first

Reply:
fedora is planning it, and even i want to push our support beyond them